src/ui/pages/CreateIdentity/useCreateIdentity.ts

```diff
@@ -14,6 +14,7 @@ import { getMessageTemplate, signWithSigner } from "@src/ui/services/identity";
 export interface IUseCreateIdentityData {
   isLoading: boolean;
   isProviderAvailable: boolean;
+  isMetamaskConnected: boolean;
```

Comment:
I think it doen't make sense now if we have to still support connecting to MM from the identities page. 
Because now we have two Different Wallet types, so in the `CreateIdentity` page if the user wants to use Metamask we try to connect first and then do the signature. 

**OR**

From the identities page we will ask the user while wallet you will like to use for creating identities, and depending on the choice we will have different scenarios: 

1. User choose Eth Wallet (Metamask for now):
- We wont need to show the user on the setup page the generated seed phrase of CK_WALLET. 
- We will connect to MM and simply follow the old way that we had before adding CK_WALLET.   
 
2. User choose CK Wallet.
- We will show the user the seed phrase from the starting page. 
- Then in the identities page we will show the user the pub. key generated from this seed phrase, as a connected wallet account. 
- In the create identity page we wont need to show both options again as well. 

Then we will need to support in the settings an option for the user to change which wallet want to use for creating identities. 
Some thing like that maybe.

Reply:
I think first option works. Second one doesn't have good UX.